agent/run

```diff
@@ -0,0 +1 @@
+java -jar target/remoting-kafka-agent-1.0-SNAPSHOT-jar-with-dependencies.jar -name test -master http://localhost:9090 -kafkaURL 127.0.0.1:9092
```

Comment:
Maybe makes sense to add comment that the file is for manual testing.
I would also suggest adding the extension (".sh") or a [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) line

Reply:
I agree with Oleg on both of these remaining points: 1) a comment in the file that it's intended for manual testing, and 2) a shebang line.